src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -417,5 +430,10 @@ def concat_obs_dict(obs_data, append_obs_data):
         help='turn on quality control georeality checks',
         default=False, action='store_true', required=False)
 
+    optional.add_argument(
+        '-l', '--localspectralwidth',
```

Comment:
```suggestion
        '-lsw', '--localspectralwidth',
```

let's maybe have this be a slightly longer short name